src/webgl/material.js

```diff
@@ -184,16 +184,19 @@ p5.prototype.createShader = function(vertSrc, fragSrc) {
 };
 
 /**
- * Like <a href="#/createShader">createShader()</a>, but with
- * a default vertex shader so that only a fragment shader is needed.
- * Creates a new <a href="#/p5.Shader">p5.Shader</a> object.
+ * Creates a new <a href="#/p5.Shader">p5.Shader</a> object with only a fragment shader, intended for creating image effects on the canvas.
+ * Like <a href="#/createShader">createShader()</a>, but with a default vertex shader included.
  *
- * The fragment shader is given a uniform, or variable, called `tex0`.
+ * Note:
+ * - The fragment shader is given a uniform, or variable, called `tex0`.
```

Comment:
Can we change the language to clarify that the default input is a texture called tex0?

Something like: "The fragment shader is provided with a single texture input uniform called `tex0`"